apps/minifront/src/components/ibc/ibc-in/ibc-in-form.tsx

```diff
@@ -1,28 +1,27 @@
-import { Button } from '@penumbra-zone/ui/components/ui/button';
-import { LockClosedIcon } from '@radix-ui/react-icons';
-import { InterchainUi } from './interchain-ui';
-import { useStore } from '../../../state';
-import { ibcInSelector } from '../../../state/ibc-in';
+import { ChainDropdown } from './chain-dropdown';
+import { CosmosWalletConnector } from './cosmos-wallet-connector';
+import { AssetsTable } from './assets-table';
+import { IbcChainProvider } from './chain-provider';
+import { useRegistry } from '../../../fetchers/registry';
+import { IbcInRequest } from './ibc-in-request';
 
 export const IbcInForm = () => {
```

Comment:
Before we had a single `<InterchainUi />` nested inside. Didn't really make sense, so moved this up a level.